sdk/loadtesting/load-testing-rest/generated/clientDefinitions.ts

```diff
@@ -0,0 +1,408 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
```

Comment:
why this file is newly added instead of moving from outside of generated folder ?

Reply:
this package was released when emitter was not fully ready